### PR TITLE
allow pipeline to continue even when Jenkins stage fails

### PR DESCRIPTION
**Pull request by peterpd**
Link to issue -- https://github.com/spinnaker/spinnaker/issues/685

### CODE REVIEW

**Comment by peterpd** [comment]
@tomaslin : PTAL of the code changes.
thanks,
peter

**Comment by tomaslin** [comment]
LGTM - see Chris' comments on what to call FAILED_CONTINUE

File: orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy [hunk 1]
```diff
@@ -124,7 +124,12 @@ class TaskTasklet implements Tasklet {
     if (result.status == ExecutionStatus.TERMINAL) {
       def shouldFailPipeline = (stage.context.failPipeline == null ? true : stage.context.failPipeline) as String
       def terminalStatus = Boolean.valueOf(shouldFailPipeline) ? ExecutionStatus.TERMINAL : ExecutionStatus.STOPPED
+      def shouldContinuePipeline = (stage.context.continuePipeline == null ? false : stage.context.continuePipeline) as String
       result = new DefaultTaskResult(terminalStatus, result.stageOutputs, result.globalOutputs)
+      if (shouldContinuePipeline == "true") {
```
Comment (ajordens) [inline orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy]: Seems weird that we coerce this to a string and not a boolean?

**Comment by peterpd** [comment]
@ajordens : agree with using boolean instead of string to check continuePipeline. 
code modified.

**Comment by anotherchrisberry** [comment]
@ajordens is incommunicado this week, but I believe we're good here - give it one more rebase, accept our apologies, and we'll get this merged!

**Comment by peterpd** [comment]
rebased.
thanks,
peter

**Pull request merged**